### ISSUE TITLE
FEATURE: Add 15:00 Timer

### ISSUE THREAD

**Issue by codereport** [codereport/scrabble#31]

**Comment by NikosRoukoudis** [comment]
Hi there!
I would like to try adding a 15-minute timer to the game.
Should I open a pull request?

**Comment by codereport** [comment]
Absolutely : ) The 15 minutes should only count down when your turn is playing : )

**Comment by NikosRoukoudis** [comment]
Of course, each player has 15 minutes to finish his/her turn. Shortly I would open a pull request

**Comment by codereport** [comment]
Well what I meant by this was that the computer doesn't need 15:00, it plays automatically / instantly.
